.github/workflows/ci.yaml

```diff
@@ -41,6 +41,15 @@ jobs:
       with:
         ruby-version: ${{ matrix.ruby-version }}
         bundler-cache: true
+    - name: Wait for db
+      run: |
+          COUNT=1
+          while ! pg_isready ; do
+            echo "waiting for postgres to start"
+            COUNT=$(($COUNT + 1))
+            [[ $COUNT -lt 20 ]] || break
+            sleep 1
+          done
```

Comment:
I don't think this will solve the actual problem we are seeing though, because it fails the health checks before it even gets here.  I think I might just bump the --health-retries to 10

Reply:
But thanks for this...going to keep the PR open in case we want to reuse parts of this.